Update quickstart-python.md

fujikosu opened the pull request:
client for batch AI and client for resource management should be different variables otherwise these scripts don't work when you run these in a row and it is very confusing for beginners.
So updated variable names and fixed one print sentence because it didn't work too.

azurecla commented:
Hi __@Juliako__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (Juliako). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

fujikosu committed:
Update quickstart-python.md

client for batch AI and client for resource management should be different variables otherwise these scripts don't work when you run these in a row and it is very confusing for beginners.
So updated variable names and fixed one print sentence because it didn't work too.
